api/Program.cs

```diff
@@ -1,30 +1,116 @@
 using api.Interfaces;
 using api.Services;
+using api.Models;
+using System;
+using System.Threading.Tasks;
+using System.Configuration;
+using System.Collections.Generic;
+using System.Net;
+using Microsoft.Azure.Cosmos;
 
-var builder = WebApplication.CreateBuilder(args);
+public class Program
+{
+    private static readonly string EndpointUri = "https://kemcowiki.documents.azure.com:443/";

```

Comment:
This stuff is fine here for now - long term we will want to move it into a settings file that we can use when deploying.

Reply:
sounds good. was just taken from code examples anyway